### ISSUE TITLE
building on ios error: property 'webViewEngine' not found on object of type 'FCMPlugin *'

### ISSUE THREAD

**Issue by RezaRahmati** [fechanique/cordova-plugin-fcm#48]
Hi

When I try to build for ios I get this error
```
Plugins/cordova-plugin-fcm/FCMPlugin.m:100:15: error: property 'webViewEngine' not found on object of type 'FCMPlugin *'
1>          [self.webViewEngine evaluateJavaScript:notifyJS completionHandler:nil];
```

I am using cordova-ios@4.2.0
cordova 5.4.1
latest version of  fcm plugin

**Comment by fechanique** [comment]
may be update to Cordova 6

**Comment by RezaRahmati** [comment]
@fechanique I have upgraded my cordova to 6.3 no build error is gone
the problem is when I open the app it asks for allowing push-notification and then it closes with out any error, when I remove the plugin, app is working fine

**Comment by fechanique** [comment]
check you have added to your project the .plist file

**Comment by fechanique** [comment]
Maybe check the bundle name or your certifications

2016-07-26 20:21 GMT+02:00 Reza Rahmati <notifications@github.com>:

**Comment by RezaRahmati** [comment]
Do I need to enter appId and teamId in app definition in fcm developer console?

**Comment by fechanique** [comment]
You need to use the same boundle (com.app.test....) in your certificates
and in your xcode project, and everywhere you are asked to put that

2016-07-26 20:55 GMT+02:00 Reza Rahmati <notifications@github.com>:

**Comment by RezaRahmati** [comment]
Yes, bundle name is the same, I was asking because in fcm console there are two other optional textboxes for app Id and Team Id

**Comment by RezaRahmati** [comment]
@fechanique can you mention what is the ecosystem versions of your working environment, I mean version of cordova, ios-cordova, npm, plugin, ...

**Comment by fechanique** [comment]
I`m not sure, but usually I work with the latest versions

2016-07-26 21:06 GMT+02:00 Reza Rahmati <notifications@github.com>:

**Comment by RezaRahmati** [comment]
I've created  a simple app with no other plugins, just fcm
I've tested on ios with Cordova 6.3 , cordova 6.2 and cordova 6.0
All is the same and app crashes at the beginning
